tests/lib/rules/prefer-use-template-ref.js

```diff
@@ -263,30 +276,41 @@ tester.run('prefer-use-template-ref', rule, {
       filename: 'options-api-only-refs.vue',
       code: `
       <template>
-        <label ref="label">
+        <label ref="labelElem">
           Name:
           <input v-model="name" />
         </label>
-        <button
+        {{ loremIpsum }}
       </template>
       <script>
         import { ref } from 'vue';
         export default {
           name: 'NameRow',
+          props: {
+            defaultLabel: {
+              type: String,
+            },
+          },
           data() {
             return {
-              label: ref(),
+              label: ref(this.defaultLabel),
+              labelElem: ref(),
```

Comment:
Are `ref`s actually supported by Vue in Option API's `data` like this?

Reply:
It doesn't break but Vue acts like the `ref` isn't there at all, it will simply unwrap the value. I wanted to prove that it works even in such a case, should I remove the test?

Reply:
Okay, then it's fine I guess. Thanks for the clarification!